sysid-application/src/main/native/cpp/view/Analyzer.cpp

```diff
@@ -751,28 +751,17 @@ void Analyzer::DisplayFeedbackGains() {
   // Come back to the starting y pos.
   ImGui::SetCursorPosY(beginY);
 
-  auto ShowLQRParam = [this](const char* text, double* data, float min,
-                             float max, bool power = true) {
-    float val = *data;
-    ImGui::SetNextItemWidth(ImGui::GetFontSize() * 5);
-    ImGui::SetCursorPosX(ImGui::GetFontSize() * 9);
-
-    if (ImGui::SliderFloat(text, &val, min, max, "%.1f",
-                           ImGuiSliderFlags_None |
-                               (power ? ImGuiSliderFlags_Logarithmic : 0))) {
-      *data = static_cast<double>(val);
-      UpdateFeedbackGains();
```

Comment:
Locally, changing the parameters doesn't change the gains. I think it's missing an `UpdateFeedbackGains` call somewhere.

Reply:
fixed